docs/content/config/advanced/mail-mta-sts.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Advanced | MTA-STS'
```

Comment:
```suggestion
title: 'Best Practices | MTA-STS'
hide:
  - toc # Hide Table of Contents for this page
```

Reply:
Please also rename this file to `mta-sts.md`, and move the file to `docs/content/config/best-practices/`.

You'll need to also add an entry here for it to be added to the actual docs:

https://github.com/docker-mailserver/docker-mailserver/blob/d988d8a8d1be8bd07c271eaa9911f15f5cd033c7/docs/mkdocs.yml#L124-L126

Add this entry after the DKIM one:

```yaml
- 'MTA-STS': config/best-practices/mta-sts.md
```